src/NuGet.Core/NuGet.Logging/project.json

```diff
@@ -20,7 +20,8 @@
     "net45": {},
     "dnxcore50": {
       "dependencies": {
-        "System.Runtime": "4.0.20-beta-23109"
+        "System.Runtime": "4.0.20-beta-23109",
+        "System.Collections.Concurrent": "4.0.11-beta-23516"
```

Comment:
Use 4.0.10-beta-23109  to match NuGet.Configuration and other projects in nuget.core.